Some data engineering joaks (#156)
wesbos/dad-jokes: readme.md

**A:** No idea. They just ransomware.

---

> **Q:** Why did the data engineer get fired from his job as a hat maker?

Reviewer: ```suggestion
**Q:** Why can’t data engineers become hat makers?
```

> **A:** He could only make two thirds of a CAP!

Reviewer: ```suggestion
**A:** They can only guarantee two thirds of a CAP!
```

**A:** No idea. They just ransomware.

---

**Q:** Why did the data engineer get fired from his job as a hat maker? 

> **A:** He could only make two thirds of a cap!

Reviewer: I don't get it 🤷‍♂

Author: Maybe this could help? https://en.wikipedia.org/wiki/CAP_theorem

---

**Q:** Why was the hippie such a good database administrator?

> **A:** He knew so much about acid

Reviewer: I think this could be improved:

```
**Q:** How did the hippie learn about database transactions?

**A:** By taking ACID
```

Author: oo good edit
